core/internal/leet/rightsidebar.go

```diff
@@ -0,0 +1,233 @@
+package leet
+
+import (
+	"fmt"
+	"time"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/wandb/wandb/core/internal/observability"
+)
+
+const (
+	systemMetricsHeader = "System Metrics"
+
+	// Sidebar content padding (accounts for borders and internal spacing).
+	sidebarContentPadding = 3
+
+	// Default grid height for system metrics when not calculated from terminal height.
+	defaultSystemGridHeight = 40
+
+	// Mouse click coordinate adjustments for border/padding.
+	mouseClickPaddingOffset = 1
```

Comment:
These constants don't look right-sidebar-specific; not sure if this is the best place to define them.

Reply:
renamed and moved to styles.go